app/components/Button.tsx

```diff
@@ -1,4 +1,4 @@
-import { FC, ReactNode } from "react";
+import { ReactNode } from "react";
```

Comment:
I don't believe this is necessary here. We can discuss this more in office hours if you will be joining us tomorrow.